examples/instrument.py

```diff
@@ -80,8 +80,7 @@ def add_command_line_args_to_variant_spec(variant_spec, command_line_args):
 
 
 def generate_experiment_kwargs(variant_spec, command_line_args):
-    # TODO(hartikainen): Allow local dir to be modified through cli args
-    local_dir = '~/ray_results'
+    local_dir = command_line_args.local_dir
```

Comment:
This looks good to me. Just for my own future reference: If I recall correctly, the reason I originally didn't do this was that there's some edge case where not using the default `~/ray_results` local directory made the results get synced to weird places. I can't figure out now what the problem was, so maybe it's been fixed e.g. on the Ray side.